src/content/quickstart/kind/_index.md

```diff
@@ -7,14 +7,24 @@ weight: 100
 ## KIND
 
 [KIND](https://github.com/kubernetes-sigs/kind) is a tool to run local Kubernetes clusters inside Docker container nodes.
+Please note that docker must be installed and running on your computer.
 
-Submariner provides scripts that deploy 3 Kubernetes clusters locally, 1 broker and 2 data clusters with the Submariner dataplane components deployed on all the clusters.
+Submariner provides (via [Shipyard](https://github.com/submariner-io/shipyard)) scripts that deploy 3 Kubernetes clusters locally - 1 broker and 2 data clusters with the Submariner dataplane components deployed on all the clusters.
 
-Please note that docker must be installed and running on your computer.
-To deploy the setup, clone [submariner repo](https://github.com/submariner-io/submariner) and run
+### Deploying a Basic Environment
+
+If you wish to deploy just a basic 3 cluster environment, clone the [shipyard repo](https://github.com/submariner-io/shipyard) and run
+
+`make deploy`
```

Comment:
Thanks for this. I have been creating this by editing e2e.sh file and then running make ci e2e